netty/src/main/java/io/grpc/netty/NettyClientStream.java

```diff
@@ -50,12 +50,25 @@
 import io.netty.handler.codec.http2.Http2Stream;
 import io.netty.util.AsciiString;
 
+import java.util.IdentityHashMap;
+import java.util.Map;
+import java.util.function.Supplier;
```

Comment:
@carl-mastrangelo It's amazing how many insights I get during long rides as well.  No review is ever a waste, rather I look at them as a natural progression :)  Though is `Supplier` also in Java 6?  Maybe that check might not be necessary for this part of the code.